install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -328,8 +328,11 @@ data:
 {{- if .Values.global.kubeConfigPath }}
   k8s-kubeconfig-path: {{ .Values.global.kubeConfigPath | quote }}
 {{- end }}
-{{- if not (eq .Values.global.cni.chainingMode "none") }}
-  # Chaining mode was enabled, disabling health checking
+{{- if (eq .Values.global.cni.chainingMode "portmap") or (eq .Values.global.cni.chainingMode "none") }}
```

Comment:
This doesn't seem to work.

It must be:
```
{{- if or (eq .Values.global.cni.chainingMode "portmap") (eq .Values.global.cni.chainingMode "none") }}
```

Reply:
Done.